wp-content/themes/missilethreat/assets/_scss/components/_newsletter.scss

```diff
@@ -44,6 +44,10 @@
       margin-bottom: rem(16px);
     }
 
+    &__button {
+      height: fit-content;
```

Comment:
I'm not actually sure you can use `fit-content` on the height property. o.O If this works, then great!

Reply:
Yup! [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content)